### PR TITLE
Disable SSL in development

### PR DESCRIPTION
**Pull request by tommotaylor**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -52,5 +52,8 @@
   config.action_mailer.perform_deliveries = false
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.default :charset => "utf-8"
+
+  # Set to false so rails server works in development. True in all other envs
```
Comment (kjdchapman) [inline config/environments/development.rb]: I'd drop this comment - strikes me as:
```ruby
# perform the service
service.perform
```

**Comment by kjdchapman** [comment]
From the Dockerfile:
```Dockerfile
RUN RAILS_ENV=production rails assets:precompile
```

File: config/environments/development.rb (hunk 1 above)
Comment (tommotaylor) [reply config/environments/development.rb]: Fair

**Comment by kjdchapman** [comment]
My mistake - the `production` flag in the Dockerfile is just for asset precompilation.

**Pull request merged**